crates/rolldown_common/src/inner_bundler_options/types/es_module.rs

```diff
@@ -0,0 +1,28 @@
+#[cfg(feature = "deserialize_bundler_options")]
+use schemars::JsonSchema;
+#[cfg(feature = "deserialize_bundler_options")]
+use serde::Deserialize;
+
+#[derive(Debug, Default)]
+#[cfg_attr(
+  feature = "deserialize_bundler_options",
+  derive(Deserialize, JsonSchema),
+  serde(rename_all = "kebab-case", deny_unknown_fields)
+)]
+pub enum EsModuleType {
+  Always,
+  Never,
+  #[default]
+  IfDefaultProp,
```

Comment:
Would be nice to add some document here.

```rust
/// What does this affects
pub enum EsModuleType {
   /// What does this do. 
   Always,
   /// What does this do.
  Never,
   /// What does this do. And why it's default and special.
  #[default]
  IfDefaultProp,
}
```

You could reference the rollup's documents.